django/text_extractor/utils.py

```diff
@@ -284,7 +286,24 @@ def create_searchable_pdf(input_file, add_header):
         new_pdf_page = PdfReader(ocr_overlay)  # changed
         output.add_page(new_pdf_page.pages[0])
 
-    return output, all_text
+    # Calculate token counts
+    input_token_count = len(all_text.split())
+    output_token_count = len(all_text.split())
+    embed_token_count = 0  # Assuming no embedding tokens for this example
+
+    # Create cost objects
+    usd_cost = 0
+    if input_token_count > 0:
+        c1 = Cost.objects.new(cost_type="doc-ai-read", count=input_token_count)
```

Comment:
costs based on texts' tokens

Reply:
The document intelligence API is costed by number of pages. You only need to create the 1 cost object:

```
num_pages = # Number of pages you sent to the API
cost = Cost.objects.new(cost_type="doc-ai-read", count=num_pages)
```

There is an example of this in `process_engine.py`

(There are no tokens (input, output or embedding) being used because you are not calling an LLM).